scripts/targetwindow.lic

```diff
@@ -47,14 +49,19 @@ module TargetWindow
     old_targets = GameObj.targets
     _respond("<closeDialog id='TargetWindow'/><openDialog type='dynamic' id='TargetWindow' title='Targets' target='TargetWindow' scroll='manual' location='main' justify='3' height='68' resident='true'><dialogData id='Targets'></dialogData></openDialog>")
     loop {
-      if GameObj.targets.length > 0
+      targets = GameObj.targets.select { |t| !(t.noun =~ @grasp_arms || t.name =~ /^animated /) }
```

Comment:
Moved the logic to strip unwanted things to the top as it was getting a bit messy with needing to check for more things further down with the addition of the animate check. I also have not tested a bard to know if this matches (think it does) but doubt they would want to see their weapon here anyway.

Reply:
Oh forgot the mean reason for this was without we would get "targets: 0" if there were only excluded targets in the room rather than then the "no targets" text